File: C#/PokerProtocol/Commands/Game/TableClosedCommand.cs
using System;
using System.Collections.Generic;
using System.Text;
using EricUtility;
using EricUtility.Networking.Commands;

namespace PokerProtocol.Commands.Game
{
    public class TableClosedCommand : AbstractCommand
    {
        public static string COMMAND_NAME = "gameTABLE_CLOSED";

        public TableClosedCommand(StringTokenizer argsToken)
        {
        }

        public TableClosedCommand()
        {
        }
    }
}

File: trunk/JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands2/game/TableClosedCommand.java
package bluffinmuffin.protocol.commands2.game;

import java.util.StringTokenizer;

import bluffinmuffin.protocol.commands2.AbstractCommand;

public class TableClosedCommand extends AbstractCommand
{
    @Override
    protected String getCommandName()
    {
        return TableClosedCommand.COMMAND_NAME;
    }
    
    public static String COMMAND_NAME = "gameTABLE_CLOSED";
    
    public TableClosedCommand(StringTokenizer argsToken)
    {
    }
    
    public TableClosedCommand()
    {
    }
}
